resources/quality/imade3d_jellybox/generic_petg_0.4_medium_2-fans.inst.cfg

```diff
@@ -4,14 +4,14 @@ name = Medium
 definition = imade3d_jellybox
 
 [metadata]
+setting_version = 4
 type = quality
-material = generic_petg_imade3d_jellybox_0.4_mm_2-fans
-weight = 0
 quality_type = normal
-setting_version = 4
+weight = 0
+material = generic_petg
+variant = 0.4 mm 2-fans
 
 [values]
-adhesion_type = skirt
```

Comment:
Adhesion type can be set per extruder since it has a resolve function.

Reply:
Fixed in c99fca089